Makefile

```diff
@@ -349,8 +352,36 @@ $(PROTOC):
 	unzip -oq ${PROTOC_ZIP} -d $(CACHE) 'include/*'; \
 	rm -f ${PROTOC_ZIP})
 
+grpc-gateway:
+	@echo "Installing protoc-gen-grpc-gateway..."
+	@rm -f $@
+	@curl -o "${GOBIN}/protoc-gen-grpc-gateway" -L \
```

Comment:
Let's install stuff into `$(CACHE_BIN)` now.

Reply:
Yeah I tried it, but `protoc-gen-grpc-gateway` will be ran by `protoc` internally and we cannot set path for it. It's generally looking from installed binaries. That's the reason, we need to install `protoc-gen-grpc-gateway` globallly

Reply:
I see.  Okay, thanks for trying.

Can you try modifying path in the `Makefile`?  If it doesn't work no problem.

```
export PATH = $(PATH):$(CACHE_BIN)
```